apps/synapse-portal-framework/src/pages/FileEntityPage/FileEntityPage.tsx

```diff
@@ -38,34 +40,37 @@ function FileEntityPage() {
     )
   }
   return (
-    <Container>
-      <DetailsPageLayout>
-        <DetailsPageMenu menuSections={fileEntityPageSections} />
-        <Stack
-          sx={{
-            flex: 1,
-            gap: '40px',
-          }}
-        >
-          {entityBundle?.rootWikiId && (
-            <CollapsibleSection title="Wiki" id="wiki">
-              <MarkdownSynapse
-                ownerId={entityId}
-                wikiId={entityBundle?.rootWikiId}
+    <>
```

Comment:
nit: could use this semantic HTML tag
```suggestion
    <article>
```